### DISCUSSION TITLE
analyticsdoc/Q42023/concepts/Chap1_Overview_of_HCL_ZAO

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#9] [Q&A]
# analyticsdoc/Q42023/concepts/Chap1_Overview_of_HCL_ZAO

HCL ZAO V2.1.1 collects raw data on products and their usage on the mainframe, and then downloads collected raw data to a website, subsequently this data is uploaded to containers for further ...

https://help.hcltechsw.com/analyticsdoc/Q42023/concepts/Chap1_Overview_of_HCL_ZAO.html

**Comment by garimashringi1** [comment]
Test- Remove the depreciated URL: http://hclpnpsupport.hcltech.com

**Comment by garimashringi1** [comment]
Test reply-2

**Comment by garimashringi1** [comment]
Test comment-2

**Reply by garimashringi1** [reply]
Test Reply-3
